File: rust_with_atcoder/atcoder_practice/abc330_a/another_langs/330a.py
import sys


def main() -> None:
    data = list(map(int, sys.stdin.buffer.read().split()))
    n = data[0]
    l = data[1]
    a_list = data[2:2 + n]

    count = sum(1 for a in a_list if a >= l)
    print(count)


if __name__ == "__main__":
    main()

File: rust_with_atcoder/atcoder_practice/abc330_a/another_langs/330a.php
<?php
$tokens = preg_split('/\s+/', trim(stream_get_contents(STDIN)));
$tokens = array_map('intval', $tokens);

$n = $tokens[0];
$l = $tokens[1];
$aList = array_slice($tokens, 2, $n);

$count = 0;
foreach ($aList as $a) {
    if ($a >= $l) {
        $count++;
    }
}

echo $count . PHP_EOL;
